cli.js

```diff
@@ -17,4 +17,14 @@ var cli = meow({
 	string: ['_']
 });
 
-cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite});
+try {
+	cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite}, function (err) {
```

Comment:
You're right, but we still don't want to remove the stack from actual errors, only from the user errors we throw, so I still think we should use the above snippet and mark our errors appropriately.